### DISCUSSION TITLE
compare all files in folder with subfolders to another?

### DISCUSSION THREAD

**Discussion by Camineet** [WinMerge/winmerge#1483] [Q&A]
Hi, is there a way to compare a folder with many subfolder's contents to another folder with many subfolders with contents, and determine which contents are not included in one or the other?

So far, I can only understand how to use this tool to compare the contents of a directory to another directory. But I want to know if the contents of one directory are found anywhere in another directory or its subfolders.

Thanks.

**Comment by sdottaka** [comment]
I may not understand exactly what you want to do, but would the following URL be helpful?
https://github.com/WinMerge/winmerge/discussions/1415
